framework/filters/HostControl.php

```diff
@@ -0,0 +1,166 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yii\filters;
+
+use Yii;
+use yii\base\Action;
+use yii\base\ActionFilter;
+use yii\web\NotFoundHttpException;
+
+/**
+ * HostControl provides simple control over requested host name.
+ *
+ * This filter provides protection against 'host header' attacks, allowing action execution only for specified
+ * host names.
```

Comment:
`'host header' attacks` should have a link that describes more what this is about. e.g. https://www.acunetix.com/vulnerabilities/web/host-header-attack